modules/kubernetes-addons/cert-manager-istio-csr/locals.tf

```diff
@@ -0,0 +1,22 @@
+locals {
```

Comment:
We don't need to use any locals here - everything can be defined directly within the helm addon module. you can use https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/kubernetes-addons/chaos-mesh/main.tf as a reference